app/main.py

```diff
@@ -17,3 +17,14 @@
 collection_of_coins = {1, 2, 25}
 
 # write your code here
+sorted_variebles = {
```

Comment:
The variable name `sorted_variebles` is misspelled. It should be `sorted_variables`.